pkg/logentry/stages/timestamp.go

```diff
@@ -15,6 +15,9 @@ const (
 	ErrEmptyTimestampStageConfig = "timestamp stage config cannot be empty"
 	ErrTimestampSourceRequired   = "timestamp source value is required if timestamp is specified"
 	ErrTimestampFormatRequired   = "timestamp format is required"
+	Unix                         = "Unix"
```

Comment:
nitpick: a new line might help discern error vs const.

Reply:
done and done